tcms/testplans/static/testplans/js/search.js

```diff
@@ -165,7 +170,22 @@ export function pageTestplansSearchReadyHandler () {
             processing: '<div class="spinner spinner-lg"></div>',
             zeroRecords: 'No records found'
         },
-        order: [[1, 'asc']]
+        order: [[2, 'asc']]
+    })
+
+    // header “select all”
+    $('#resultsTable thead').on('change', '#select-all', function () {
```

Comment:
Not liking this in the header that much. 

Can you move the "Select All" + the "Clone Selected" buttons into the button box above the table? 

See how the rest of the buttons are defined. For icon, use fa-code-fork icon to match existing UI. 

The order should be
`[ ] <clone-selected> <excel> <.... the rest of the buttons>`.


Note: (a follow up improvement can be made to the rest of these table buttons to operate only on the current selection if present.